cmd/moco-backup/cmd/root.go

```diff
@@ -45,6 +49,26 @@ func makeS3Bucket(bucketName string) (bucket.Bucket, error) {
 	if commonArgs.usePathStyle {
 		opts = append(opts, bucket.WithPathStyle())
 	}
+	if len(commonArgs.caCertFilePath) > 0 {
+		caCertFile, err := os.ReadFile(commonArgs.caCertFilePath)
+		if err != nil {
+			return nil, err
+		}
+		caCertPool, err := x509.SystemCertPool()
+		if err != nil {
+			return nil, err
+		}
+		if ok := caCertPool.AppendCertsFromPEM(caCertFile); !ok {
+			return nil, fmt.Errorf("failed to add ca cert")
+		}
+		transport := http.DefaultTransport.(*http.Transport).Clone()
+		transport.TLSClientConfig = &tls.Config{
+			RootCAs: caCertPool,
+		}
```

Comment:
This comment might be a kind of imaginary fears.
Please ignore if you consider this overanxious.

`DefaultTransport.TLSClientConfig` is not set in the current implementation, so you can (and you must) assign a new `Config`.
It is not guaranteed, however, that the field will always be nil in the future.

```suggestion
		if transport.TLSClientConfig == nil {
			transport.TLSClientConfig = &tls.Config{}
		}
		transport.TLSClientConfig.RootCAs = caCertPool
```